### ISSUE TITLE
🐛 Bug Report: The field value cannot be updated when the attribute name is purely numeric

### ISSUE THREAD

**Issue by Hu-Wentao** [appwrite/appwrite#3256]
### 👟 Reproduction steps

1. create an collection, like "test".
2. add some attribute , and set purely numeric name, like 1, 2, 3
![image](https://user-images.githubusercontent.com/35894003/169529024-e9088d9a-d248-42ac-bfc1-8a2ceea7cbf2.png)
3. insert an documnet.
4. update some documnet attribute
5. get error `Invalid document structure: Unknown attribute: "2"`
![image](https://user-images.githubusercontent.com/35894003/169529169-138cb4f0-be21-4403-8a9d-1d9a6ee73d57.png)

As you can see, I didn't create an attribute named `2`.

if add attribute named `2`, will also be prompted for `Unknown attribute: "4"`

### 👍 Expected behavior

update some attribute value.

### 👎 Actual Behavior

Get the error prompt that there is no association.

### 🎲 Appwrite version

Version 0.13.x

### 💻 Operating system

Windows

### 🧱 Your Environment

Windows10
Docker-WSL2

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
Still a problem on Appwrite version 1.2.0. The appwrite logs are:

```
appwrite  | [Error] Timestamp: 2023-02-08T19:33:53+00:00
appwrite  | [Error] Method: PATCH
appwrite  | [Error] URL: /v1/databases/:databaseId/collections/:collectionId/documents/:documentId
appwrite  | [Error] Type: Appwrite\Extend\Exception
appwrite  | [Error] Message: Invalid document structure: Unknown attribute: "2"
appwrite  | [Error] File: /usr/src/code/app/controllers/api/databases.php
appwrite  | [Error] Line: 2343
```

It's odd the error says 

> Unknown attribute: "2"

when 2 isn't passed...

**Comment by stnguyen90** [comment]
I think this fix needs to happen in the [utopia-php/database repo](https://github.com/utopia-php/database). Maybe there's something wrong with the [Structure](https://github.com/utopia-php/database/blob/3e48e346041cb51dafb0fc0eb1eba56989077a46/src/Database/Validator/Structure.php#L194) validator.

**Comment by abnegate** [comment]
This is now fixed in Appwrite 1.5 🥳
